src/ui/exile_modal.py

```diff
@@ -0,0 +1,47 @@
+import discord
+from services.exile_service import exile_user
+from util import calculate_time_delta
+from .helper import create_modal_embed
+
+
+class ExileModal(discord.ui.Modal):
+    def __init__(self, user: discord.Member) -> None:
+        super().__init__(title=f"Exile User {user.display_name}")
+        self.user = user
+
+    duration = discord.ui.TextInput(label="Exile Duration")
+
+    reason = discord.ui.TextInput(
+        label="Exile Reason",
+        style=discord.TextStyle.long,
+        placeholder="Why is the user exiled",
+        max_length=300,
+        required=True,
+    )
+
+    async def on_submit(self, interaction: discord.Interaction):
+
+        async with create_modal_embed(
+            interaction,
+            "Exile User",
+            user=self.user,
+            duration=self.duration.value,
```

Comment:
we should be safe and error check duration since they are required now

Reply:
I've marked the field as required on the modal, which has good UI checks and won't let you submit if either field is blank